e2e-tests/e2e-tests.ts

```diff
@@ -88,7 +88,7 @@ async function verifyJourney (matchUser?: boolean) {
 async function eidasEnabledJourney (selectEidas?: boolean, unsignedAssertions?: boolean) {
   await page.goto(host, { waitUntil: 'networkidle2' })
   await page.click('a.button-start')
-  await page.waitForSelector('h1.heading-large')
+  await page.waitForSelector('h1.govuk-heading-l')
```

Comment:
Can we just wait for `'hi'` - the class is an entirely presentational detail about which we should not care.

Reply:
`h1` is too generic. There is a redirect/transistion page (`Continue to next step`). e2e tests will to try click on a specific button which doesn't exist on the transition page which has `h1`.

Reply:
I tried `h1`. e2e tests failed.